nameko/cli/main.py

```diff
@@ -45,8 +45,11 @@ def _replace_env_var(match):
 
 
 def _env_var_constructor(loader, node):
-    value = loader.construct_scalar(node)
-    return ENV_VAR_MATCHER.sub(_replace_env_var, value)
+    raw_value = loader.construct_scalar(node)
+    value = ENV_VAR_MATCHER.sub(_replace_env_var, raw_value)
+    new_tag = loader.resolve(yaml.ScalarNode, value, (True, False))
```

Comment:
The name doesn't make anything clearer to me. But then again, neither does digging through the pyyaml source (http://pyyaml.org/browser/pyyaml/trunk/lib3/yaml/resolver.py#L140)

@Trex-Boolat can you explain what that third argument does?